### PR TITLE
Update resourcemanager service poms to use correct dependencies

### PR DESCRIPTION
**Pull request by srnagar**
This PR fixes the list of dependencies in resourcemanager and resourcemanagerhybrid directory.

### CODE REVIEW

**Review by alzimmermsft — APPROVED** [review]

**Review by weidongxu-microsoft — APPROVED** [review]
What does azure-sdk-template do?

File: sdk/resourcemanagerhybrid/pom.xml [hunk 1]
```diff
@@ -59,9 +59,69 @@
       </modules>
       <dependencies>
         <dependency>
-          <groupId>com.azure</groupId>
-          <artifactId>azure-sdk-template</artifactId>
-          <version>1.1.1234</version> <!-- {x-version-update;com.azure:azure-sdk-template;dependency} -->
+          <groupId>com.azure.resourcemanager</groupId>
+          <artifactId>azure-resourcemanager</artifactId>
+          <version>1.0.0-hybrid</version>
```
Comment (JimSuplizio) [inline sdk/resourcemanagerhybrid/pom.xml:64]: @srnagar why do these have a non-standard version and no version tag?

**Review by JimSuplizio — CHANGES_REQUESTED** [review]
The resourcemanager tags are all set to use azure-resourcemanager's version instead of the versions of their respective libraries. 
The resourcemanagerhybrid has no tags but there may be a reason for that which I've asked.

**Comment by srnagar** [comment]
@weidongxu-microsoft  the azure-sdk-template is a template for creating new client libraries. The structure of all new client libraries should be similar to the template module. It's also used to test the release pipelines and are not intended for customer use.

**Pull request merged**